Create model output endpoint (#65)
CABLE-LSM/meorg_client: docs/cli.md

This command will return an `$ANALYSIS_ID` upon success which is used in `analysis status`.

### model output create

> To start an analysis for a given model output using the files provided, execute the following command:

Reviewer: ```suggestion
To create a model output, execute the following command:
```

```shell
meorg output create $MODEL_PROFILE_ID $EXPERIMENT_ID $MODEL_OUTPUT_NAME
```

> Where `$MODEL_PROFILE_ID` and `$EXPERIMENT_ID` are found on the model profile details and corresponding experiment details pages in question. `$MODEL_OUTPUT_NAME` is the name for the newly created model output.

Reviewer: ```suggestion
Where `$MODEL_PROFILE_ID` and `$EXPERIMENT_ID` are found on the model profile and corresponding experiment details pages on modelevaluation.org. `$MODEL_OUTPUT_NAME` is a unique name for the newly created model output.
```

This command will return the newly created `$MODEL_OUTPUT_ID` upon success which is used for further analysis. It will also print whether an existing model output record was returned or a new record was created.

### model output query

Retrieve Model output details via `$MODEL_OUTPUT_ID`

```shell
meorg output query $MODEL_OUTPUT_ID
```

> This command will print the `id` and `name` of the modeloutput, and JSON representation for the remaining metadata.

Reviewer: Does it dump the JSON? Do we want it to do that?

Author: Hmm, I thought it could be useful to have all the model output details for the developer mode, so I have added a conditional check for that.
